besecure_developer_toolkit/src/vdnc.py

```diff
@@ -0,0 +1,71 @@
+"""vdnc validate"""
+import sys
+import json
```

Comment:
can we remove json from here if we are not using

Reply:
done